From 582d7cddcda260684d2bca7265294dc651bcb091 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Mon, 29 Feb 2016 16:54:05 -0800
Subject: [PATCH 55/75] tests: Stop timing t9040 (#1172675)

parted doesn't have any control over how long this takes, there is no
reason to consider this a parted bug if it takes longer than expected.

Resolves: rhbz#1172675
(cherry picked from commit fdd2e12b504c1d8a7829055cf8195d4aae4cd708)
---
 tests/t9040-many-partitions.sh | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/tests/t9040-many-partitions.sh b/tests/t9040-many-partitions.sh
index a3af5e7..4f58658 100644
--- a/tests/t9040-many-partitions.sh
+++ b/tests/t9040-many-partitions.sh
@@ -53,9 +53,7 @@ while :; do
 done
 
 # Time the actual command:
-t0=$(date +%s.%N)
 parted -m -a min -s $scsi_dev mklabel gpt $cmd u s p > out 2>&1 || fail=1
-t_final=$(date +%s.%N)
 
 i=1
 while :; do
@@ -64,14 +62,6 @@ while :; do
     printf "$i:${s}s:${e}s:${partition_sectors}s::p$i:;\n" >> exp
     test $i = $n_partitions && break; i=$((i+1))
 done
-
-# Fail the test if it takes too long.
-# On Fedora 16, this takes about 10 seconds for me.
-# With Fedora-12-era kernels, it typically took more than 150 seconds.
-$AWK "BEGIN {d = $t_final - $t0; n = $n_partitions; st = 60 < d;"\
-' printf "created %d partitions in %.2f seconds\n", n, d; exit st }' /dev/null \
-    || fail=1
-
 compare exp out || fail=1
 
 Exit $fail
-- 
2.9.3

